### ISSUE TITLE
style.css encoding problems

### ISSUE THREAD

**Issue by JBPressac** [omeka/theme-seasons#37]
Hello,

Seems that there are encoding problems in [style.css](https://github.com/omeka/theme-seasons/blob/54acd255082a3e47112d9bf37ded916d691afd7c/css/style.css#L382-L389):

```
  .pagination_previous a:after {
    content: "";
  }


  .pagination_next a:after {
    content: "";
  }

...

  #search-form button:after {
    font-family: "Font Awesome 5 Free";
    font-weight: 900;
    content: "";
...
  .asc a:after {
    content: "";
  }


  .desc a:after {
    content: "";
  }
...
  nav.top .sub-nav ul a:before {
    font-family: "Font Awesome 5 Free";
    font-weight: 900;
    content: "";
    margin-right: 0.5em;
  }
  nav.top .sub-nav ul ul {
    padding-left: 1em;
  }
  nav.top .sub-nav li {
    display: block;
    list-style-type: none;
    margin: 0;
    padding: 0;
  }
  nav.top .sub-nav a:focus, nav.top .sub-nav a:hover {
    background-color: rgba(0, 0, 0, 0.05);
    display: block;
  }
  nav.top [aria-haspopup=true]:after {
    font-family: "Font Awesome 5 Free";
    font-weight: 900;
    content: "";
    margin-left: 0.5em;
    display: inline-block;
    width: 6px;
  }
  nav.top [aria-haspopup=true].open:after {
    content: "";
```

**Comment by zerocrates** [comment]
Are you seeing actual rendering problems associated with any of this?

Those are places specifying which iconfont characters to use, so they're supposed to be "weird" private-use characters that don't render as anything in source form.

**Comment by JBPressac** [comment]
I don't know where those icons are supposed to be displayed in the HTML pages and I did not found this problem because of HTML rendering problems. I did noticed it because the text editor I used to edit this file replaced the weird characters by a question mark. Besides [those strange characters are in the GitHub source file](https://github.com/omeka/theme-seasons/blob/54acd255082a3e47112d9bf37ded916d691afd7c/css/style.css#L382-L389). 

[In previous commit](https://github.com/omeka/theme-seasons/blob/c398d888cbf36e238081ba24446452d7b6914903/css/style.css#L278-L279), the unicode characters corresponding to "weird characters" are correctly displayed (for instance,  is displayed `\f0d9`). Which make me thing that the commit ['Edit theme styles against new public styles.'](67e4aa1e527a82724e2e3ae88c8a9563196636e5) introduced an encoding problem in the source style.css file.
